vcr/signature/ldutils.go

```diff
@@ -28,12 +28,27 @@ import (
 	"net/url"
 )
 
+// JSONLDContextsConfig contains config for json-ld document loader
+type JSONLDContextsConfig struct {
```

Comment:
I'd rather have all config-related structs in the `config.go` of the particular engine, to keep things tightly coupled and isolated.  So we don't have config-implementation specifics (`koanf` tags) all over the place.